### PR TITLE
Cns secgrp

### PR DESCRIPTION
**Pull request by iamemilio**
@tzumainn 

No longer adds CNS security group to HOT stack if there are no CNS created (openshift_openstack_num_cns variable is set to 0).

https://bugzilla.redhat.com/show_bug.cgi?id=1613438

### CODE REVIEW

File: roles/openshift_openstack/templates/heat_stack.yaml.j2 [hunk 1]
```diff
@@ -1093,9 +1094,11 @@ resources:
             - { get_resource: flat-secgrp }
 {% else %}
             - { get_resource: node-secgrp }
+{% if openshift_openstack_num_cns > 0 %}
             - { get_resource: cns-secgrp }
+{% endif %}
 {% if openshift_use_kuryr|default(false)|bool %}
-          pod_secgrp:
+         pod_secgrp:
```
Comment (tzumainn) [inline roles/openshift_openstack/templates/heat_stack.yaml.j2]: Thanks for the PR! Quick question - is this space removal intentional? The original indent looks correct to me... ?

**Comment by tzumainn** [comment]
Looks good! I think if you just return the missing whitespace on the line I commented on, this will be good to merge.

File: roles/openshift_openstack/templates/heat_stack.yaml.j2 (hunk 1 above)
Comment (iamemilio) [reply roles/openshift_openstack/templates/heat_stack.yaml.j2]: ah, yep. thats a typo

**Comment by sdodson** [comment]
This is only on master branch, if we intend to fix this on 3.11 and 3.10 we need to cherrypick it back there.

**Pull request merged**